insights/parsers/var_opt_mssql.py

```diff
@@ -0,0 +1,29 @@
+"""
+BadOwnershipOfVarOptMSSql - datasource ``ownership_of_var_opt_mssql``
+=====================================================================
+"""
+import json
+from insights.specs import Specs
+from insights.parsers import SkipException
+from .. import CommandParser, parser
+
+
+@parser(Specs.ownership_of_var_opt_mssql)
+class BadOwnershipOfVarOptMSSql(CommandParser):
```

Comment:
The parser is too specific.  if we check other dir/file ower/group, need repeat the codes.

Reply:
Maybe we create one base class for owner/group checking.  Others just inherit it.